nbs/01_Tutorials/01_tutorial.ipynb

```diff
@@ -16,7 +16,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Perhaps you can clarify the relationship bw markdown front matter and raw cell front matter?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
<p>I'd prefer to leave that for the Frontmatter reference or an explanation. I'm trying to keep this sorta "top-down", so only explaining little bits at a time to keep the user engaged and flowing quickly through the tutorial, then linking out for more</p>